tests/access_test.go

```diff
@@ -53,6 +54,57 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.BeforeTestCleanup()
 	})
 
+	Describe("With the kubevirt-controller service account", func() {
+		table.DescribeTable("should verify permissions on resources are correct for kubevirt-controller", func(resource string) {
+
+			controllerVerbs := make(map[string]string)
```

Comment:
I think this might be more readable if we pass this map to the test function instead of using a switch?

Reply:
Thanks. Ginkgo was new to me. I think I got to understand you suggestion. Just want to spell out my understanding and sync with you. 

The caller side would look like this? 
```
table.Entry("given a kv", map[string]string{"list":"yes", "get":"yes", "create":"yes",.......}, "kubevirt")
```

Also, using slice here can reduce the code, but takes an effort to derive the mapping. I do not see many usage of slices yet. What's your thought?

Reply:
yup! your example is exactly what I was thinking!